staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -245,21 +242,23 @@ func (o *AuditWebhookOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.ConfigFile, "audit-webhook-config-file", o.ConfigFile,
 		"Path to a kubeconfig formatted file that defines the audit webhook configuration."+
 			" Requires the 'AdvancedAuditing' feature gate.")
-	fs.StringVar(&o.Mode, "audit-webhook-mode", o.Mode,
```

Comment:
We need to remind users that `audit-webhook-mode` is depercated. And they should use xxx insted.

Reply:
This behavior should also be recorded in the release note, I think.

Reply:
I'm not sure what's the policy for removing flags

@sttts Could you please advise on that?

Reply:
Flags should be deprecated for at least 1 release prior to removal. I suggest just keeping this flag as-is, even though it's slightly inconsistent with the log flags.